rfc-101-local-development-data.md

```diff
@@ -0,0 +1,51 @@
+# Next steps with local development data
+
+## Summary
+
+Update the way we manage local development data, promote using the
+`govuk data` tool, and setup the automation to make this possible.
+
+## Problem
+
+Presently, many developers on GOV.UK copy data from the S3 database
+backups bucket on to there local machine, and use this for local
+development.
+
+This use of realistic data can aid local development, but the data
+used in the real services and local development are different use
+cases, and there are aspects of the process that make local
+development more difficult.
+
+Particularly, you don't necessarily need to have near-exact copies of
+data for all services on GOV.UK for local development. Downloading
+data for all services takes significant time and uses large amounts of
+storage space.
+
+The [`govuk data` tool][govuk-data-docs] can help with these problems,
```

Comment:
I agree it's inconsistent, but the `govuk` command is itself a script [1], and for the `govuk data` command, this provides some important functionality in the govuk-puppet dev VM. Particularly it has some awareness of rbenv [2].

Having that space, rather than an underscore allows wrapping of many commands with a single script, in a very similar way to other tools, Git for example.

1: https://github.com/alphagov/govuk-guix/blob/master/bin/govuk
2: https://github.com/alphagov/govuk-guix/blob/master/bin/govuk#L78-L83